presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/FifoQueue.java

```diff
@@ -74,4 +74,10 @@ public boolean isEmpty()
     {
         return delegate.isEmpty();
     }
+
```

Comment:
Change commit message to:

Change UpdateablePriorityQueue to implement Iterable